src/common/Section/style.js

```diff
@@ -1,24 +1,42 @@
-import styled from "styled-components";
+import styled, { css } from "styled-components";
 
 export const Container = styled.section`
   display: grid;
   align-items: center;
   justify-items: center;
   grid-template-columns: repeat(4, 324px);
   grid-gap: 24px;
+
   @media (max-width: ${({ theme }) => theme.breakpoints.tablet}px) {
     grid-template-columns: repeat(2, 324px);
   }
+
   @media (max-width: ${({ theme }) => theme.breakpoints.mobileMax}px) {
     grid-template-columns: 1fr;
   }
+
+  ${(props) =>
+    props.people &&
+    css`
+      grid-template-columns: repeat(6, 208px);
+
+      @media (max-width: ${({ theme }) => theme.breakpoints.tablet}px) {
+        grid-template-columns: repeat(4, auto);
+      }
```

Comment:
Do you think there is no need to add a grid-gap for tablets?

Reply:
There is grid-gap - 24px. I just don't change it